### ISSUE TITLE
ssri vulnerabilities

### ISSUE THREAD

**Issue by jrramsaransing** [SAP/karma-ui5#302]
I'm getting the following vulnerabilities:

```
karma-ui5@2.3.3 requires ssri@^6.0.1 via a transitive dependency on cacache@12.0.4
karma-ui5@2.3.3 requires ssri@^6.0.0 via a transitive dependency on make-fetch-happen@5.0.2
karma-ui5@2.3.3 requires ssri@^6.0.1 via a transitive dependency on pacote@9.5.12
```

Can you please update the dependencies that uses `ssri`, to a minimum of 8.0.1:
```
ssri 5.2.2-8.0.0, fixed in 8.0.1, processes SRIs using a regular expression which is vulnerable to a denial of service. Malicious SRIs could take an extremely long time to process, leading to denial of service. This issue only affects consumers using the strict option.
```

https://github.com/advisories/GHSA-vx3p-948g-6vhq

**Comment by matz3** [comment]
This comes via [@ui5/project](https://github.com/SAP/ui5-project) which uses pacote in v9.x. So it needs to be solved there, e.g. by upgrading to the most recent version v11.x.

**Comment by codeworrior** [comment]
@matz3 But doesn't this conflict with the problems mentioned in https://github.com/SAP/ui5-project/pull/299 ?

**Comment by RandomByte** [comment]
> @matz3 But doesn't this conflict with the problems mentioned in [SAP/ui5-project#299](https://github.com/SAP/ui5-project/pull/299) ?

I think we can test this again. npm v7 also uses pactote v11 internally, so maybe these issues have been resolved in the meantime.

However, certainly the easiest solution would be if the ssri fix would be downported and released as a v6 patch: https://github.com/npm/ssri/issues/18  
This would allow us to consume the fix simply by updating our lockfiles.

**Comment by matz3** [comment]
npm/ssri#18 has been resolved. The fix has been ported to ssri v6.0.2 which is used once your lock-files have been updated. No actions required in karma-ui5 or ui5-project.
